src/components/Board.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import Square from "./Square"
+
+const Board = (props) => {
+  const [currentPlayer, setCurrentPlayer] = useState('Current player: X');
```

Comment:
Why do you need to put `Current player:` text to state?

Reply:
I have a field showing whose turn it is to play? this text is shown there.